tools/interop_matrix/client_matrix.py

```diff
@@ -53,7 +53,7 @@ def should_build_docker_interop_image_from_release_tag(lang):
 # Dictionary of default runtimes per language
 LANG_RUNTIME_MATRIX = {
     "cxx": ["cxx"],  # This is actually debian8.
-    "go": ["go1.8", "go1.11", "go1.16", "go1.19"],
+    "go": ["go1.x"],
```

Comment:
Would this now effect the previous releases? Just double verifying. 

Seems like this is how the rest of the languages are set up. So not too concerned.

Reply:
It will not effect the previous releases since they've provided the runtime version explicitly (not depending on the default specified here). The following code uses the runtime provided with the release if available and falls back to the default:
https://github.com/grpc/grpc/blob/d110960d567f203c7928522a4f45b4c5306ae4af/tools/interop_matrix/client_matrix.py#L35-L41